discoverecs.py

```diff
@@ -527,6 +531,7 @@ def main():
     arg_parser.add_argument("--directory", required=True)
     arg_parser.add_argument("--interval", default=60)
     arg_parser.add_argument("--default-scrape-interval-prefix", default="1m")
+    arg_parser.add_argument("--cluster", default=None)
```

Comment:
See above, it'd be better if this is a list. 

It'd also be better to be explicit and call it `--cluster-arns`, right now it's unclear whether we mean cluster name in the same account, or the full arn.

I've just made a change to simplify the log at the start and add an option which uses a list if you want to try to follow that example.

Reply:
That looks very helpful. I will rebase my change on the latest HEAD to be able to use all of this. I hope it doesn't mess up the review